protoc-gen-grpc-gateway/main.go

```diff
@@ -34,6 +34,7 @@ var (
 	repeatedPathParamSeparator = flag.String("repeated_path_param_separator", "csv", "configures how repeated fields should be split. Allowed values are `csv`, `pipes`, `ssv` and `tsv`.")
 	allowPatchFeature          = flag.Bool("allow_patch_feature", true, "determines whether to use PATCH feature involving update masks (using google.protobuf.FieldMask).")
 	allowColonFinalSegments    = flag.Bool("allow_colon_final_segments", false, "determines whether colons are permitted in the final segment of a path")
+	standalone                 = flag.Bool("standalone", false, "generates standalone gateway package, which imports target service")
```

Comment:
```suggestion
	standalone                 = flag.Bool("standalone", false, "generates a standalone gateway package, which imports the target service package")
```

Reply:
Sure, thank you for the corrections. I will take into account the comments and prepare changes

Reply:
I've added notes about the new functionality in readme.md, made generation of examples with the standalone flag set and made rebase to the v2 branch.